source/reference/minio-mc/mc-support-perf.rst

```diff
@@ -0,0 +1,208 @@
+===================
+``mc support perf``
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support perf
+
+Description
+-----------
+
+Use the :mc-cmd:`mc support perf` command to analyze the performance of an object, network, or drive.
```

Comment:
```suggestion
Use the :mc-cmd:`mc support perf` command to analyze the performance of the S3 read/write, network IO, or drive read/write.
```

It's not really "performance of an object", it's S3 read/write performance. We should make that change throughout this page.